src/utils.ts

```diff
@@ -53,6 +53,35 @@ const promptLayerApiRequest = async (body: TrackRequest) => {
   }
 };
 
+const promptLayerAllPrompts = async (params?: Pagination) => {
+  const url = new URL(`${URL_API_PROMPTLAYER}/rest/prompts`);
+  Object.entries(params || {}).forEach(([key, value]) =>
+    url.searchParams.append(key, value)
+  );
+  try {
+    const response = await fetch(url, {
+      method: "GET",
+      headers: {
+        "Content-Type": "application/json",
+        "X-API-KEY": getApiKey(),
+      },
+    });
+    const data = await response.json();
+    if (response.status !== 200) {
+      console.warn(
+        `WARNING: While fetching all prompts, PromptLayer had the following error: ${JSON.stringify(
+          data
+        )}`
+      );
+    }
+    return data;
+  } catch (e) {
+    console.warn(
+      `WARNING: While fetching all prompts, PromptLayer had the following error: ${e}`
```

Comment:
Again, "prompt templates" not prompts.

And make this match the other messages in wording